ruskeyverbs/migrations/0005_auto_20180224_2126.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.0.2 on 2018-02-25 03:26
+
+from django.db import migrations
+from django.utils import timezone
+import random
+import datetime
+
+
+def populate_user(apps, schema_editor):
```

Comment:
Is this method just for development purposes to get dummy data in there to work with or is it more part of the actual application? If it's just for dev, it probably technically shouldn't be in a migration. It could just be a separate .py file that you run by hand if you need to...

Reply:
Yes, this migration is for populating dummy data for development purposes. I think I'll just delete it before merging.